assignments/assignment-8-rest/assignment-7.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.7</title>
+    </head>
+    <body>
+        <form>
+            <h2>Photo Contest Entry</h2>
+            <label for="fullNameInput">Full Name</label>
+            <input
+                type="text"
+                id="fullNameInput"
+                name="fullName"
+                placeholder="Full Name"
+                required
+            />
+            <label for="emailInput">Your Email Address</label>
+            <input
+                type="email"
+                id="emailInput"
+                name="yourEmail"
+                placeholder="Email"
+                required
+            />
+            <label for="fileInput">Upload a Photo</label>
+            <input
+                type="file"
+                id="fileInput"
+                name="uploadPhoto"
+                accept="image/*"
+                required
+            />
+            <label for="agreeCheckbox"
+                >Agree to the Contest Terms and Conditions</label
+            >
+            <input
+                type="checkbox"
+                id="agreeCheckbox"
+                name="agreeToTheContest"
+                required
+            />
+            <br />
+            <br />
```

Comment:
It is better to use div instead of br in this case. Like:
"div" label input "/div"